airflow/Dockerfile

```diff
@@ -8,9 +9,24 @@ RUN apt-get update && apt-get install -y python3-pip \
     libcurl4-gnutls-dev \
     librtmp-dev \
     python3-dev \
-    libpq-dev
-
-RUN python3 -m pip install PyGreSQL argcomplete pycurl
+    python3-setuptools \
+    libpq-dev \
+    gcc \
+    build-essential \
+    g++ \
+    git-all \
+    unixodbc-dev \
+    apt-utils \
+    apt-transport-https \
+    debconf-utils \
+    vim \
+    telnet
```

Comment:
Instead of bundling in too many packages, consider moving them to optional `requirements.txt`, which user can fill in.
If needed, update ReadMe with details

Reply:
Isn't requirements.txt a Python thing?

Reply:
Yes. And all these packages are being installed using pip.  
I should have included Line #11. It looks like this..
```
RUN python3 -m pip install PyGreSQL argcomplete pycurl
    python3-setuptools \
    .....
```